fuzzers/035-iob-ilogic/generate.py

```diff
@@ -4,6 +4,16 @@
 from prjxray import verilog
 import json
 
+# Set to true to enable additional tags useful for tracing bit toggles.
+DEBUG_FUZZER = False
+
+
+def bitfilter(frame, word):
```

Comment:
I always wonder where this information comes from other than that they keep the bits from being solved nicely.

Reply:
You comment is about right.  The filter in this case is removing bits that are potentially correlated, but not believed to be directly part of the solution.  Concretely, INT_[LR] bits are in the (0, 25), but there are other bits that need to be filtered.